src/rpc/blockchain.cpp

```diff
@@ -1138,6 +1138,10 @@ bool FillScRecordFromInfo(const uint256& scId, const CSidechain& info, CSidechai
             sc.push_back(Pair("wCertVk", HexStr(info.creationData.wCertVk)));
             sc.push_back(Pair("customData", HexStr(info.creationData.customData)));
             sc.push_back(Pair("constant", HexStr(info.creationData.constant)));
+            if (info.creationData.wMbtrVk.is_initialized())
+            	sc.push_back(Pair("wMbtrVk", HexStr(info.creationData.wMbtrVk.get())));
+            else
+                sc.push_back(Pair("wMbtrVk", HexStr(libzendoomc::ScVk{std::vector<unsigned char>(SC_VK_SIZE,'\0')})));
```

Comment:
Maybe a "NOT INITIALIZED" string would save chars

Reply:
Done